Makefile

```
@@ -63,67 +27,43 @@ TIMESTAMP := $(shell date '+%Y%m%d%H%M%S')
 .PHONY: test
-test: setup-envtest ginkgo ocm-transfer-demo ## Run all tests
-	@KUBEBUILDER_ASSETS="$(shell $(SETUP_ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" $(GINKGO) -r -cover --fail-fast --require-suite -covermode count --output-dir=$(BUILD_PATH) -coverprofile=solar.full.coverprofile $(testargs)
-	@cat solar.full.coverprofile | grep -v /solar/api > solar.coverprofile
+test: $(SETUP_ENVTEST) $(GINKGO) ocm-transfer-demo ## Run all tests
+	OCM=$(OCM) \
+	KUBEBUILDER_ASSETS="$(shell $(SETUP_ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" \
+	$(GINKGO) -r -cover --fail-fast --require-suite -covermode count --output-dir=$(BUILD_PATH) -coverprofile=solar.full.coverprofile $(testargs)
+	@grep -v /solar/api solar.full.coverprofile > solar.coverprofile
 
 .PHONY: test-e2e
 test-e2e: manifests ## Run the e2e tests. Expected an isolated environment using Kind.
```

checkmake (0.3.2)

[warning] Target body for "test-e2e" exceeds allowed length of 5 lines (7).

(maxbodylength)